### PR TITLE
Lup launcher setting from lvl

### PR DESCRIPTION
**Pull request by aronwk-aaron**
Get setting from the lvl file instead of cdclient

### CODE REVIEW

File: dGame/dComponents/RocketLaunchLupComponent.cpp [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
-
-	// get the lup worlds from the cdclient
-	std::string query = "SELECT * FROM LUPZoneIDs;";
-	auto results = CDClientDatabase::ExecuteQuery(query);
-	while (!results.eof()) {
-		// fallback to 1600 incase there is an issue
-		m_LUPWorlds.push_back(results.getIntField(0, 1600));
-		results.nextRow();
+	std::string zoneString = GeneralUtils::UTF16ToWTF8(m_Parent->GetVar<std::u16string>(u"MultiZoneIDs"));
+	std::stringstream ss(zoneString);
```
Comment (EmosewaMC) [inline dGame/dComponents/RocketLaunchLupComponent.cpp:9]: Believe this is unnecessary.  There should be a tool in general utils to split a string by a delimiter ans return a vector of the values.

**Review by EmosewaMC — CHANGES_REQUESTED** [review]

File: dGame/dComponents/RocketLaunchLupComponent.cpp (hunk 1 above)
Comment (aronwk-aaron) [reply dGame/dComponents/RocketLaunchLupComponent.cpp:9]: There are but none return the right type

**Comment by EmosewaMC** [comment]
Resolved requested change in discord.  What tests have been run on this?

**Comment by aronwk-aaron** [comment]
> Resolved requested change in discord. What tests have been run on this?

Tested going to all LUP worlds and they worked as expected

**Pull request merged**